### PR TITLE
Remove Matomo configuration

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### Description

Remove Matomo configurations for TOC

### CODE REVIEW

File: config/i18n-tasks.yml [hunk 1]
```diff
@@ -169,3 +169,4 @@ ignore_unused:
   - decidim.verifications.omniauth.authorizations.new.success
   - decidim.verifications.omniauth.errors
   - decidim.verifications.omniauth.errors.anti_affinity
+  - layouts.decidim.cookie_warning.*
```
Comment (eliegaboriau) [inline config/i18n-tasks.yml:172]: Shouldn't you remove the locales key rather than adding this line ?

**Review by eliegaboriau — APPROVED** [review]
Looks good to me except for the locales

File: config/i18n-tasks.yml (hunk 1 above)
Comment (Quentinchampenois) [reply config/i18n-tasks.yml]: Yes I updated !

File: app/views/decidim/pages/_standalone.html.erb [hunk 2]
```diff
@@ -1,6 +1,7 @@
+
 <div class="wrapper">
   <div class="row column">
-    <div class="text-center">
+    <div class="page-title-wrapper">
```
Comment (paulinebessoles) [inline app/views/decidim/pages/_standalone.html.erb:4]: It works perfectly, thanks @Quentinchampenois! 

I just have one feedback, could you please just come back to the original file class, as the titles are now displaying left, and we want them centered as before? 

```suggestion
    <div class="text-center">
```

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**